### PR TITLE
Adds Suit Sensors

### LINKED ISSUE #4885: Suit Sensors
-Add the suit sensor levels for jump suits.

Off
Tells them nothing. You don't even show up on the monitor.

Binary Life Signs
Tells them if you are alive or deceased. The name that shows up on the monitor is the same one on the ID that is being worn. If no ID is worn, you will show up as 'Unknown'.

Vital Tracker
Tells them what your vital statistics are. Separates damage by Suffocation, Toxin, Burn, and Brute.

Tracking Beacon
The maximum level. Tells them your co-ordinate location.

-Add the crew monitoring console to display this info for all players.

### PR DESCRIPTION
**Pull request by MaxIsJoe**
closes #4885

CL: [New] All outfits now come with suit sensors embedded into them to spy on crew members and their vitals.
CL: [New] Added functionality for the medical console.

### CODE REVIEW

File: UnityProject/Assets/Scripts/UI/Objects/Medical/MedicalCon/GUI_MedicalConsoleEntry.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+		[SerializeField] private Color healthyColor;
+		[SerializeField] private Color dangerColor;
+		[SerializeField] private Color deadColor;
+
+		public void SetValues(MedicalTerminal.HealthInfo info)
+		{
+			label.MasterSetValue($"{info.Info}");
+			if (info.Mode == SuitSensor.SensorMode.OFF)
+			{
+				image.color = unknownColor;
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/UI/Objects/Medical/MedicalCon/GUI_MedicalConsoleEntry.cs]: This won't network?

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/UI/Objects/Medical/MedicalCon/GUI_MedicalConsoleEntry.cs]: i know, but eh, what do i do?

Comment (Bod9001) [reply UnityProject/Assets/Scripts/UI/Objects/Medical/MedicalCon/GUI_MedicalConsoleEntry.cs]: you make the colour changing component synchronised thingy, be able to specify the component it changes the colour for, and Then add an extra layer and put the text and the colour on each layer separately

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/UI/Objects/Medical/MedicalCon/GUI_MedicalConsoleEntry.cs]: what color changing component thingy
what extra layer and how

Comment (Bod9001) [reply UnityProject/Assets/Scripts/UI/Objects/Medical/MedicalCon/GUI_MedicalConsoleEntry.cs]: is there anything for actually networking Colour currently?

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/UI/Objects/Medical/MedicalCon/GUI_MedicalConsoleEntry.cs]: nope

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/UI/Objects/Medical/MedicalCon/GUI_MedicalConsoleEntry.cs]: I've decided to just not add background color changes as it ruins visibility

**Review by Bod9001 — APPROVED** [review]

**Pull request merged**